assets/js/modules/analytics/components/settings/SettingsForm.stories.js

```diff
@@ -54,26 +68,147 @@ function Template() {
 
 export const WithGA4andUASnippet = Template.bind( null );
 WithGA4andUASnippet.storyName = 'UA and GA4 tag snippet switches';
+WithGA4andUASnippet.scenario = {
```

Comment:
As a number of these stories had become broken over time, I made a few additions to the VRT suite.